2.4/root/usr/bin/run-mongod

```diff
@@ -74,26 +74,25 @@ if [ -z "${MONGODB_REPLICA_NAME-}" ]; then
     usage
   fi
   # Run the MongoDB in 'standalone' mode
-  if [ ! -f ${MONGODB_DATADIR}/.mongodb_datadir_initialized ]; then
-    # Create MongoDB users and restart MongoDB with authentication enabled
-    # At this time the MongoDB does not accept the incoming connections.
-    mongod $mongo_common_args & #--bind_ip 127.0.0.1 --quiet >/dev/null &
-    wait_for_mongo_up
+  mongod $mongo_common_args &
+  wait_for_mongo_up
+  js_command="db.system.users.count({'user':'admin'})"
+  if [ "$(mongo admin --quiet --eval "$js_command")" == "1" ]; then
```

Comment:
It is the same as in current state.  What if database is not initialized and server is down - resetting passwd fails. Mongod should be running and it is running before this command. If `mongo admin` fails because server is down something went wrong so scripts fails in every case!

Now scripts check that mongod is running (wait_for_mongo_up) and then it assumes that it is running and do not check it every time - in that case mongod would fail in every line of code (each command should have `|| true`). So with your worry mongodb scripts would be completely wrong. And I hope it is not needed to check mongod status in every command.